packages/components/src/theme/section.scss

```diff
@@ -112,3 +112,7 @@
 .error-msg {
     color: $brand-danger;
 }
+
+.success-msg {
```

Comment:
I believe you can use `.text-primary` instead of adding this class. We could also get rid of `.error-msg` and use `.text-danger` (but we should probably defer that because it's used all over IIRC). More info at the [bootstrap docs](https://getbootstrap.com/docs/3.3/css/#helper-classes).

Reply:
good call. I removed `success-msg` and used `text-success` instead. I also replaced a couple usages of `error-msg` with `text-danger`. I'll add a TODO to the scss for removing error-msg once we know all usages are gone.